ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/Common/tasks/RedHat.yml

```diff
@@ -136,17 +136,17 @@
   with_items: "{{ Java_NOT_RHEL6_PPC64 }}"
   when:
     - not (ansible_distribution_major_version == "6" and ansible_architecture == "ppc64")
-    - not (ansible_distribution_major_version == "8")
+    - not (ansible_distribution_major_version | int >= 8)
 
 - name: Install Java when RedHat 6 on ppc64
   package: "name={{ item }} state=latest"
   with_items: "{{ Java_RHEL6_PPC64 }}"
   when: (ansible_distribution_major_version == "6" and ansible_architecture == "ppc64")
 
-- name: Install Java when RedHat 8
+- name: Install Java when RedHat 8 and RedHat 9
   package: "name={{ item }} state=latest"
   with_items: "{{ Java_RHEL8 }}"
-  when: (ansible_distribution_major_version == "8")
+  when: (ansible_distribution_major_version | int >= 8)
```

Comment:
Are these 3 cases still what we want?
From https://github.com/adoptium/infrastructure/blob/894d84829c15361afe29e72dc66387a4be5e986a/ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/Common/vars/RedHat.yml#L95
```
Java_RHEL8:
  - java-1.8.0-openjdk-devel

Java_NOT_RHEL6_PPC64:             # Not RHEL8 either
  - java-1.7.0-openjdk-devel
  - java-1.8.0-openjdk-devel

Java_RHEL6_PPC64:
  - java-1.7.0-ibm-devel
  - java-1.8.0-ibm-devel
```
What is java 7 and 8 used for?

Reply:
@AdamBrousseau, I am not sure. Should we bump it to 17 or 21?

Reply:
As far as I can tell it's been there since the beginning. Java is needed for 2 purposes that I know of.
- Connecting agents to jenkins.
- Bootjdk for compiling java.

Since Jenkins now requires jdk17 to run agents, and jdk11 was required 2 years ago, I don't think leaving this at java8 for jenkins connections is necessary.
For bootjdk requirement, the build scripts will pull the needed jdk during the build process so having a permanent install of 8 is not necessary in my opinion.

I would want an opinion from Adopt/@sxa before we remove it completely though.

Reply:
As far as the boot JDK is concerned I don't want the builds pulling it down dynamically in most cases - that's mostly just a fallback mechanism. Other than pulling down the source, the builds should be able to run without network access.

I suspect those java-1.8.0* packages are no longer required, at least by Temurin, so it would likely be safe to remove them.

Reply:
Yes I think that should be safe. It will hopefully be obvious if some part of the process requires it, in which case it's easy to back it out again :-)

Can the 1.7 version be installed from the RHEL9 repositories?

Reply:
@sxa is there another part of the PB installing java for use by the jenkins agent that Adopt is running?
We have an internal Semeru role that installs 21 for this.

Reply:
Yep. The adoptopenjdk_install role: .https://github.com/adoptium/infrastructure/blob/028773c140a974f741d6b4ab5a78f381f2df7a68/ansible/playbooks/AdoptOpenJDK_Unix_Playbook/main.yml#L131 
It's extracting as a tarball and doesn't set any default on the system to point to it, so the Jenkins agent is pointed directly to it under /usr/lub/jvm when it's started